services/frontend/src/i18n/en_CA.json

```diff
@@ -343,6 +343,14 @@
   "search.advanced.general.title": "General info",
   "search.advanced.skill.title": "Skills and talent",
   "search.empty.query": "Refine your search criterias to start finding talent!",
+  "search.filter.name": "Input Name",
+  "search.filter.classification": "Select Classification",
+  "search.filter.location": "Select Location",
+  "search.filter.branch": "Select Branch",
+  "search.filter.skills": "Select Skills",
+  "search.filter.mentorship.skills": "Select Mentorship Skills",
+  "search.filter.any.mentors": "Search for Mentors?",
+  "search.filter.exfeeder": "Check for EX-feeder",
```

Comment:
To keep consistency within the application, usually nothing is title case because of the Government standards, could you change it to sentence case?

Reply:
Fixed.